plugins.json

```diff
@@ -1849,6 +1849,20 @@
         "date": "2023-08-25T10:26:10.17939-07:00",
         "sha512sum": "fc50088e5ba72dce117f7a9bce9bb31811aa66ef9c2ea5e303312d6b085daa8955470830da7faa77a432d0c406712d432b00286502e5090753397eb186551584",
         "requires": ">=22.10.6"
+      },
+      {
+        "version": "0.6.0",
```

Comment:
Nextflow will always pull the lastest one. Is it really necessary to add 0.6.0?

Reply:
Ah well, will remove to speed things up.